File: readsnowflafe/first.py
import snowflake.connector

# Menghubungkan ke Snowflake
conn = snowflake.connector.connect(
    user='SILVIO',
    password='S1LV1Oooo',
    account='rf60339.ap-southeast-1'
)

# Membuat kursor
cur = conn.cursor()

# Menjalankan query
cur.execute("SELECT * FROM SNOWFLAKE_SAMPLE_DATA.TPCDS_SF100TCL.CATALOG_PAGE")

# Mendapatkan hasil
for row in cur:
    print(row)

# Menutup kursor dan koneksi
cur.close()
conn.close()

File: akses sql dengan R/readsnowfalf/first.r
library(DBI)
library(odbc)

# Membuat koneksi
con <- dbConnect(odbc::odbc(),
                    Driver = "SnowflakeDSIIDriver",
                    Server = "rf60339.ap-southeast-1.snowflakecomputing.com",
                    UID = "SILVIO",
                    PWD = "S1LV1Oooo",
                    Database = "SNOWFLAKE_SAMPLE_DATA",
                    Schema = "TPCDS_SF100TCL",
                    Warehouse = "COMPUTE_WH",
                    Role = "ACCOUNTADMIN")

# Menjalankan query
result <- dbGetQuery(con, "SELECT * FROM CATALOG_PAGE")

# Menampilkan hasil
print(result)

# Menutup koneksi
dbDisconnect(con)
